spec/fixtures/files/datahashes/core/exhibition1.json

```diff
@@ -26,19 +26,19 @@
     "exhibitionPersonPersonUlan": "We go",
     "exhibitionPersonOrganizationLocal": "breakup",
     "exhibitionPersonOrganizationUlan": "Again",
-    "exhibitionPersonRole": "Educator;Public programs coordinator;",
+    "exhibitionPersonRole": "Educator^^^^Public programs coordinator",
```

Comment:
Changed order to match new order of exhibitionPerson values

Changed to subgroup value delimiter because these are multiple Persons listed inside a workingGroup, and the workingGroup can also be repeated. The following xpath structure in the XML indicates you have a group/subgroup situation: 

`workingGroupList/workingGroup/exhibitionPersonGroupList/exhibitionPersonGroup`